data/sea/51-piano.h

```diff
@@ -1,13 +1,13 @@
 #include "50-chord.h"
 
-typedef void error_t;
+typedef void piano_error_t;
 typedef struct piano piano_t;
 
 int64_t piano_max_count (
     piano_t *piano
   );
 
-error_t piano_lookup (
+piano_error_t piano_lookup (
```

Comment:
This can be `error_t` and you can delete the `typedef`, piano uses the anemone error